### DISCUSSION TITLE
documentação inglês?

### DISCUSSION THREAD

**Discussion by dorathoto** [maritaca-ai/maritalk-api#5] [General]
Porque a documentação é em inglês se o maritaca.ai é especificamente para o pt-br ? quem for utilizar para inglês acredito que utilizaria outra solução como Openai, Roberta, Llama, etc;

Nada contra documentação em inglês é até padrão. Mas para um projeto que quer vestir a camiseta verde amarelo sempre, pode não ser necessário.

Só para complementar o notebook python está em pt-br

https://colab.research.google.com/drive/13tieiQdQqYDQGHI8aLtlqoBWMpJ2elyo?usp=sharing

**Comment by rodrigo-f-nogueira** [comment]
Oi @dorathoto , gostamos da sugestão e já traduzimos o README para o português.

**Comment by peq42** [comment]
a página de chat tinha que ter versão em inglês também, para os gringos poderem conseguir usar.
Isso deveria ser bem simples de implementar

**Comment by peq42** [comment]
alguma informação sobre o que perguntei antes, sobre a pagina de chat ter versão em inglês? @rodrigo-f-nogueira 
Se os gringos não tiverem como usar aquela página, eles não irão comprar a API, já que não conseguirão testar o modelo antes

**Reply by rodrigo-f-nogueira** [reply]
Oi @peq42, está nos planos, sim.. obrigado por lembrar!
